README.md

```diff
@@ -255,3 +255,59 @@ The command used to shutdown the system.
 Set the environment variable `RIZZO_DEBUG` to `true`.
 
 example: `RIZZO_DEBUG=true vagrant status`
+
+# Commands
+
+## `rizzo generate`
+
+These command names are suggestions. Happy to entertain different ones.
+
+Will generate 'Vagrantfile' in your top control repo. Turn existing
+Vagrantfile into an erb.
+
+`spec/spec_helper.rb` will need at least the following.
+
+Perhaps we should manage this entire file.
+
+```ruby
+RSpec.configure do |config|
+  config.hiera_config = 'hieradata/hiera.yaml'
+end
+```
+
+## `rizzo compile_test`
```

Comment:
This functionality can wait until the next release. The first release should just handle generating the Vagrantfile